app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -84,6 +84,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 */
 		var resumeSubtractDuration = stringPreference("pref_resume_preroll", "0")
 
+		/**
+		 * Foreground Color for the Subtitles
+		 */
+		var subtitleFGColor = stringPreference("subtitleFGColor", "#ffffffff")
```

Comment:
Probably better to store the color as long (e.g. 0xffffffff would be 4294967295) and convert that on usage instead of using a string.

Reply:
Absolutely right, i wasnt too happy with the storage method to be honest, so im glad you helped there, I will make this change